Source/WebKit/UIProcess/WebScreenOrientationManagerProxy.cpp

```diff
@@ -122,15 +122,15 @@ void WebScreenOrientationManagerProxy::lock(WebCore::ScreenOrientationLockType l
     if (resolvedLockedOrientation != m_currentlyLockedOrientation) {
         bool didLockOrientation = false;
 #if ENABLE(FULLSCREEN_API)
-        if (m_page.fullScreenManager() && m_page.fullScreenManager()->isFullScreen()) {
-            if (!m_page.fullScreenManager()->lockFullscreenOrientation(resolvedLockedOrientation)) {
+        if (m_page->fullScreenManager() && m_page->fullScreenManager()->isFullScreen()) {
+            if (!m_page->fullScreenManager()->lockFullscreenOrientation(resolvedLockedOrientation)) {
```

Comment:
Seems we need a `protectedPage()` to call `fullScreenManager()` on.

Reply:
right, fixed.